index.html

```diff
@@ -15,12 +15,49 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  Meet makes it easy to connect with others face-to-face virtually and collaborate across any device.
-
-  <button class="btn btn-primary">Download v1.34.1</button>
-  <button class="btn btn-secondary">Download v1.34.1</button>
-  Download v1.3
-  What is it?
+  <header>
+    <img class="main__logo" src="assets/logo.svg" alt="Logo image">
```

Comment:
Give the whole page some width, it looks huge like this.
The rest is good.

Reply:
Done

Reply:
Great!